code/modules/station_goals/dna_vault.dm

```diff
@@ -257,8 +257,10 @@ var/list/non_simple_animals = typecacheof(list(/mob/living/carbon/monkey,/mob/li
 	switch(upgrade_type)
 		if(VAULT_TOXIN)
 			H << "<span class='notice'>You feel resistant to airborne toxins.</span>"
-			S.tox_breath_dam_min = 0
-			S.tox_breath_dam_max = 0
+			if(/obj/item/organ/lungs in H.internal_organs)
```

Comment:
It would be `if(locate(/obj/item/organ/lungs) in H.internal_organs)` then, right?